pyproject.toml

```diff
@@ -8,7 +8,7 @@ requires = [
 
 [project]
 name = "ansys-dynamicreporting-core"
-version = "0.7.0.dev0"

```

Comment:
I believe when the metapackage will be released, since now there is an official v0.7.0 release, the milestone that PyAnsys will generate for PyDynamicReporting will be 0.8.0 and not 0.7.0. I believe it would be better to remove the release 0.7.0 and let PyAnsys handle for us, what do you think?
According to the PyAnsys guidance, when we switch to full release 0.7.0, the dev release should point at the next major version, that should so be v0.8.0dev0, and create 0.7.X release during the next (2025R2) cycle

Reply:
@mariostieriansys Is this what you're suggesting (made the change)?

Reply:
@margalva partially. This solves the issue of point 2
The problem is that in a few day PyAnsys will create a milestone for the next "major release" to be used for the 25.1 metapackage.
Since the latest official release if now "0.7.0" since you released yesterday this version, the new milestone will be 0.8.0. Which is fine, but it means that there will be also a release 0.8.0 virtually equivalent to 0.7.0
Since I believe you needed to create a new release, I would have done:

1) Release 0.6.1, not 0.7.0
2) When PyAnsys would have created the milestone it would be 0.7.0
3) once the milestone is released, convert main to point to 0.8.0dev0

The trouble is that the PyAnsys Metapackage will contain 0.8.0, the EnSight/ADR distribution will have 0.7.0, that again, are virtually equivalent but not from the Python versioning point of view, and you would have to have a new OSS form to include 0.8.0 in the ADO distribution instead of 0.7.0


My suggestion is to delete the release 0.7.0, create a release 0.6.1 so that you can have a new release for the stakeholders that have requested it and finally wait for PyAnsys to create for us the next milestone that will so be 0.7.0 and not 0.8.0.

OR

When the milestone will be created, that will try to make a 0.8.0 release, I believe it should be possible for you to just close it that is equivalent to declaring "pick the latest release (0.7.0) for the metapackage and not create a new one".

I will start approving the PR because it fits with the current latest release. At this stage I believe the second option is the easiest